### ISSUE TITLE
Window Size does not remain same

### ISSUE THREAD

**Issue by YashovardhanSingh** [thenewboston-developers/Account-Manager#512]
**Bug Description**
The window size of the *TNB Account Manager*  does not remain the same as the size when it was closed previously.

**Steps to Reproduce**
1. Open TNB Account Manager
2. adjust its size
3. Close the Manager
4. Open again

**Expected behavior**
The window size should remain the same as the size when it was closed previously.

**Actual behavior**
The window size does not remain the same.

**OS and Browser**
 - OS: Windows 10

**Account Number**
38a99b8f1916eab0471ca1df26e7fe5e71d845e6ea5b233cd3d93a75097ea2d6

**Comment by YashovardhanSingh** [comment]
@angle943 Is there no Bug bounty?

**Comment by stweyel** [comment]
This BTW is not only Win related. Same behaviour on MacOS. If that's going to change the window position should be considered to be saved as well. Maybe more a question of design than a bug?

94161c7f2d143eb387aab95b1ee4d334df57a233a63bd466c8b2368148c4917d

**Comment by AbhayAysola** [comment]
@angle943 we could use [this package](https://www.npmjs.com/package/electron-window-state-manager) for window state management or implement it on our own as to not increase bloat.

**Comment by stweyel** [comment]
Will anybody be assigned for that? Would it be ok if i do a PR @angle943 ,@AbhayAysola ?

**Comment by AbhayAysola** [comment]
Sure, I'd wait for @angle943 's reply though. @stweyel

**Comment by angle943** [comment]
sure, anyone can work on this @stweyel and @AbhayAysola

**Comment by stweyel** [comment]
> sure, anyone can work on this @stweyel and @AbhayAysola 

Great! Got it already running. Need to sort out a thing and do some testing before submitting a PR.

**Comment by YashovardhanSingh** [comment]
Is this task complete?

**Comment by angle943** [comment]
Yes now it is. @YashovardhanSingh and @stweyel you guys will receive a payment soon. I've submitted a bounty request: https://github.com/thenewboston-developers/Contributor-Payments/issues/259

**Comment by manishram** [comment]
> With reference to this: https://github.com/thenewboston-developers/Contributor-Payments/issues/259?notification_referrer_id=MDE4Ok5vdGlmaWNhdGlvblRocmVhZDE4MzU4NTczNjA6MjI3OTA5MDQ%3D&notifications_after=Y3Vyc29yOjUw#issuecomment-827336712

@angle943 can you please send coins to @YashovardhanSingh and @stweyel

**Comment by angle943** [comment]
@YashovardhanSingh @stweyel terribly sorry you guys! I had the amount just sitting in the FE wallet. I've sent out the two payments!

**Comment by stweyel** [comment]
> @YashovardhanSingh @stweyel terribly sorry you guys! I had the amount just sitting in the FE wallet. I've sent out the two payments!

Thanks!

**Comment by Creatinas** [comment]
Shouldn't it also have minimum window size?
